### ISSUE TITLE
Postgres Upgrade for PQ from 9.3 to 9.6 - Staging Environment

### ISSUE THREAD

**Issue by AntonyBishop** [ministryofjustice/cloud-platform#155]
## Background

As part of a stories in the previous sprint we identified Products that were dependant on Postgres v9.3 for the purposed of upgrading them to a supported version. A list of products was identified (#85) and Product Teams were contacted (#92) to tell them about the need to upgrade and to ask what support might be required.

The Parliamentary Questions (PQ) Team confirmed that they would need assistance from Cloud Platform to carry out the upgrade on their behalf.

## Proposed user journey and Approach

1. Depends on #153  being completed.
2. Agree with PQ when Staging can be upgraded
3. Carry out upgrade from 9.3 to 9.6 using AWS Postgres upgrade console
4. PQ Team test PQ Staging instance
5. PQ Team sign-off on upgrade success and agree any monitoring period
6. If unsuccessful implement rollback procedure as be #148
7. PQ and CP investigate any errors and agree to repeat upgrade if required

## Questions / Assumptions

1. Depends on #153 
2. PQ Team have identified a contact to work with us (confirmed as Daniel Penny in Tactical Products)
3. PQ Team have confirmed that Dev, Staging and Production are using the same version of Postgres (confirmed by Andy Robb)
4. PQ application is compatible with Postgres 9.6 (confirmed by Andy Robb)
5. It is possible that we need to investigate and carry out any remedial action to ensure that an old version of Postgres is not deployed in the event of a complete rebuild. This work will be carried out under a separate story.

## Definition of done

- [x] #153  completed
- [x] agreement for PQ Team to upgrade Staging instance of Postgres
- [x] upgrade implemented
- [x] upgrade documentation updated
- [x] PQ documentation updated

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)
[List of products using Postgres 9.3](https://docs.google.com/spreadsheets/d/1iZUJ99tnPSH0SS4TE_JMNEigvjZWrMWNLgdoi8CrJ0E/edit#gid=0)

**Comment by lukaszraczylo** [comment]
Upgrade performed, waiting for confirmation from @danielatdigital
